wiki/Glossary/BeatmapID/en.md

```diff
@@ -0,0 +1,18 @@
+---
+tags:
+  - links
+---
+
+# Beatmap ID
+
+*See also: [BeatmapSetID](/wiki/Glossary/BeatmapSetID)*
+
+A *beatmapID* is a unique, incrementing number given to all uploaded difficulties of a [beatmapset](/wiki/Beatmaps/Beatmapsets). This number is shown on a beatmapset's beatmap listing page on the website. The beatmapSetID can be found in the url of a beatmapset on osu! web as the second set of numbers, or in the `.osu` difficulty file. Beatmaps will only be assigned a beatmapID once they've been submitted with the Beatmap Submission System.
```

Comment:
```suggestion
A **beatmap ID** is a unique, incrementing number given to all uploaded difficulties of a [beatmapset](/wiki/Beatmaps/Beatmapsets). This number is shown on a beatmapset's [beatmap listing page](https://osu.ppy.sh/beatmapsets) on the website. The beatmapSetID can be found in the url of a beatmapset on the osu! website as the second set of numbers, or in the `.osu` difficulty file. Beatmaps will only be assigned a beatmap ID once they've been submitted with the Beatmap Submission System
```
> The beatmapSetID can be found in the url of a beatmapset on the osu! website as the second set of numbers[...]

I would love to have an image of a beatmap's URL with a highlight over those mentioned numbers under this line. we could add a "(shown below)" after this sentence.